test/tools/locale-nav/locale-selector.test.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable import/no-unresolved */
+import { expect } from '@esm-bundle/chai';
+import sinon from 'sinon';
+import '../../../tools/locale-nav/locale-selector.js';
+
+const locales = [
+  {
+    code: 'uk',
+    path: '/uk/blog/index',
+    edit: 'https://da.live/edit#/adobecom/da-bacom-blog/uk/blog/index',
+    preview: 'https://main--da-bacom-blog--adobecom.hlx.page/uk/blog/',
+    live: 'https://main--da-bacom-blog--adobecom.hlx.live/uk/blog/',
+  },
+  {
+    code: 'au',
+    path: '/au/blog/index',
+    edit: 'https://da.live/edit#/adobecom/da-bacom-blog/au/blog/index',
+    preview: 'https://main--da-bacom-blog--adobecom.hlx.page/au/blog/',
+    live: 'https://main--da-bacom-blog--adobecom.hlx.live/au/blog/',
+  },
+  {
+    code: 'de',
+    path: '/de/blog/index',
+    edit: 'https://da.live/edit#/adobecom/da-bacom-blog/de/blog/index',
+    preview: 'https://main--da-bacom-blog--adobecom.hlx.page/de/blog/',
+    live: 'https://main--da-bacom-blog--adobecom.hlx.live/de/blog/',
+  },
+  {
+    code: 'fr',
+    path: '/fr/blog/index',
+    edit: 'https://da.live/edit#/adobecom/da-bacom-blog/fr/blog/index',
+    preview: 'https://main--da-bacom-blog--adobecom.hlx.page/fr/blog/',
+    live: 'https://main--da-bacom-blog--adobecom.hlx.live/fr/blog/',
+  },
+  {
+    code: 'kr',
+    path: '/kr/blog/index',
+    edit: 'https://da.live/edit#/adobecom/da-bacom-blog/kr/blog/index',
+    preview: 'https://main--da-bacom-blog--adobecom.hlx.page/kr/blog/index',
+    live: 'https://main--da-bacom-blog--adobecom.hlx.live/kr/blog/index',
```

Comment:
This one should remove index at the last part

Reply:
Thanks!